ckanext/data_qld_theme/templates/package/snippets/additional_info.html

```diff
@@ -10,11 +10,14 @@
   'owner_org',
   ] -%}
 
-{%- if h.data_qld_user_has_admin_access(True) == False -%}
+{%- if h.data_qld_user_has_admin_access(True) == False  or pkg_dict.get('next_update_due') is none -%}
```

Comment:
@tino097 Can you remove the `or pkg_dict.get('next_update_due') is none` condition as it should only be in the block below

Reply:
i shouldn't push that late, reverted back to original code